### PR TITLE
Add EPWW ACC new sectors

### PR DESCRIPTION
**Pull request by aerobird**
Add EPWW ACC new sectors

### CODE REVIEW

File: FIRBoundaries.dat [hunk 1]
```diff
@@ -15505,7 +15855,7 @@ RJCG|0|0|11|21.000000|125.416667|50.083333|165.000000|41.614100|138.370000
 39.227330|143.252500
 39.414300|141.910300
 39.165800|139.921700
-RJBG|0|0|7|21.004444|121.498611|30.003611|136.997500|25.504028|129.248056
```
Comment (Connorseds2002) [inline FIRBoundaries.dat:15508]: Why the removal of this when added just some time ago to fix with the new callsign

File: FIRBoundaries.dat [hunk 2]
```diff
@@ -15466,7 +15816,7 @@ RCAA|0|0|6|21.000000|117.500000|29.000000|124.000000|23.670000|121.390000
 29.000000|124.000000
 23.300000|124.000000
 21.000000|121.300000
-RJTG|1|0|26|21.000000|125.416667|50.083333|165.000000|32.541667|150.208334
```
Comment (Connorseds2002) [inline FIRBoundaries.dat:15469]: Why the change back to as it was before it was changed

**Review by Connorseds2002 — CHANGES_REQUESTED** [review]
Need to look at the some of the changes that are being removed

File: VATSpy.dat [hunk 3]
```diff
@@ -18059,7 +17942,7 @@ RKRR-S|Incheon ACC (South) - Incheon||RKRR-S
 RKDA|Daegu ACC - Incheon||RKDA
 RKDA-E|Daegu ACC (East) - Incheon||RKDA
 RKDA-C|Daegu ACC (Central) - Incheon||RKDA
-RJBG|Kobe ACC - Fukuoka||RJBG
+RORG|Naha ACC - Fukuoka||RORG
```
Comment (Connorseds2002) [inline VATSpy.dat:17945]: need to look at this as well I am not shore if the user has an old version on there end

**Comment by Connorseds2002** [comment]
@aerobird I am not shore if you have an old version on your end but there is a lot being changed for one thing as there was an update today to fix some fo the position within the japan as of pull #158

**Review by prryplatypus — CHANGES_REQUESTED** [review]
This PR has plenty of changes unrelated to EPWW's ACC. Please ensure only changes to the specified sectors are being done.

File: FIRBoundaries.dat (hunk 1 above)
Comment (aerobird) [reply FIRBoundaries.dat:15508]: My bad, used not up-to-date files

**Comment by aerobird** [comment]
> 
> @aerobird I am not shore if you have an old version on your end but there is a lot being changed for one thing as there was an update today to fix some fo the position within the japan as of pull #158

Yeah, Havent checked if there was update in a mean time

**Review by h-okon — APPROVED** [review]
Now - without changing anything but Warsaw EPWW - looks good to me.

**Review by Connorseds2002 — APPROVED** [review]
Now fine as well as only add EPWW acc new sectors and does not effect anything else

**Review by prryplatypus — APPROVED** [review]

**Comment by ThaweK** [comment]
Original author of the change here.

Why did somebody changed EPWW-S to EPWW_S and so on, if the "-" paradigm is followed anywhere else?

**Comment by prryplatypus** [comment]
I have just reverted the changes made in this PR, as after looking into it further, it does need some more tweaking before being merged. In addition to using `_` when the `-` paradigm is used everywhere else, the boundaries it draws do not coincide with the current borders in the FIR, and leave gaps at several different places. I'm sure @NelisV can give some more insight on this so a new PR can be submitted with this fixed.

**Comment by nakoale** [comment]
99790093-49b3af00-2b2c-11eb-9b56-039e64822ab7

**Comment by ThaweK** [comment]
They've been moved to join right over the EPWA, because every sector, when logged in solo, covers EPWA. rest is exactly as on the screen above, so I don't get where's further issue.

**Comment by deltazulu44** [comment]
> I have just reverted the changes made in this PR, as after looking into it further, it does need some more tweaking before being merged. In addition to using `_` when the `-` paradigm is used everywhere else, the boundaries it draws do not coincide with the current borders in the FIR, and leave gaps at several different places. I'm sure @NelisV can give some more insight on this so a new PR can be submitted with this fixed.

Where are the gaps? Which sector lines don't correspond to VACC website?

**Comment by prryplatypus** [comment]
It's not necessarily that it doesn't correspond with the sectors in the VACC's website, but that in the FIRBoundaries.dat it creates gaps with neighbouring sectors. This doesn't mean that the sectors in this PR are necessarilly wrong, but most likely that they have a different resolution than those around it. Since these gaps can lead to confusion in some cases, all that's needed is to ensure that the boundaries remain the same between neighbouring sectors; either changing it in this PR's data or modifying the neighbour's sector boundaries to match the ones submitted here. @NelisV will provide more insight soon.

**Comment by NelisV** [comment]
image
2 gaps on the border with Belarus

image
gap created on the border with kaliningrad. 

image
another weird non aligned border near Munchen Gera. 

+ the use of _ in sector names, - is the convention.

**Comment by deltazulu44** [comment]
OK. I think Kaliningrad and one of Belarus gaps are from original data. The question is whether EPWW or UMKK and UMMV are to be adjusted.

BTW. UMKK and UMMV are different countries.

**Comment by deltazulu44** [comment]
Actually or those lines of UMKK are misaligned. It's rather UMKK that requires fixing by someone.

image

**Comment by ThaweK** [comment]
Kaliningrad and Belarus lines are there since VATSpy release for 10 years or more and counting.

image
image

I don't see any issue with the Munchen. This is the result of simplified FIR/Country borders and that's how it should be.
Once again - I used original coordinates from the original FIRBoundaries.dat. These gaps never brought anyone else's attention, sadly. Instead of reverting entire change, this could be done in another release. Now, you reverted back to the version... That still has these gaps, only making people confused by appearing and disappearing sector borders over Warszawa FIR.

image


I need to rearrange not only my sectors, but also half of the neighbourhood as well, and this will take time, and I am not sure I should be responsible of. 

What software do you use for such preview? Does it allow to redraw using that? I am doing it manually, so it's laborious work. 

Another issue is, that VATSpy now tries to overwrite my manually edited FIRBoundaries.dat... Should I use older version to prevent that or is there any other way except internet disconnecting?

**Comment by NelisV** [comment]
I'm using QGIS, yes can edit it aswell, i'll dive in somewhere next week to align it properly with the bordering sectors. Should be enough information in this thread now. We can do it together, just ping me on discord and we can schedule a call. Niels#4311

**Comment by deltazulu44** [comment]
I'm using QGIS as well for other projects. Do you have some scripts to share to load / export vatspy database easily? Especially for export?

**Comment by NelisV** [comment]
@deltazulu44 you can have a look, it's not pretty, but it works :P https://github.com/NelisV/vatspy-geojson

**Comment by NelisV** [comment]
@ThaweK 

> Now, you reverted back to the version... That still has these gaps, only making people confused by appearing and disappearing sector borders over Warszawa FIR.
it's not the end of the world :), let's just try to fix it in one go instead of multiple PR's where we propagate issues.
